rpcs3/CMakeLists.txt

```diff
@@ -403,6 +401,10 @@ endif()
 
 target_link_libraries(rpcs3 xxhash yaml-cpp)
 
+find_package(ZLIB REQUIRED)
+target_include_directories(rpcs3 PUBLIC ${ZLIB_INCLUDE_DIRS})
+target_link_libraries(rpcs3 ${ZLIB_LIBRARIES})
```

Comment:
Can we use ZLIB::ZLIB HERE?

Reply:
Unfortunately, when using the submodule, `ZLIB::ZLIB` does not exist which means it will not work.